ci-operator/step-registry/assisted/baremetal/operator/catalog-publish/assisted-baremetal-operator-catalog-publish-ref.yaml

```diff
@@ -0,0 +1,62 @@
+ref:
+  as: assisted-baremetal-operator-catalog-publish
+  from: assisted-service-publish-index
+  cli: latest
+  commands: assisted-baremetal-operator-catalog-publish-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 500Mi
+  credentials:
+  - mount_path: /secrets/docker
+    name: edge-infrastructure-quay-push
+    namespace: test-credentials
+  - mount_path: /tmp/secret
+    name: agentil-quay-creds
```

Comment:
we need this in addition to ``edge-infrastructure-quay-push``?

Reply:
nope :fearful:

Reply:
I removed it